coalib/results/Diff.py

```diff
@@ -432,7 +451,7 @@ def __bool__(self):
         """
         return (self.rename is not False or
                 self.delete is True or
-                self.modified != self._file)
+                self._raw_modified() != self._file)
```

Comment:
Right.
I changed it to compare `self.modified ` with `self.original`.
Also added a test for this case.